elements/pfe-modal/src/pfe-modal.scss

```diff
@@ -0,0 +1,125 @@
+@import "../../pfe-sass/pfe-sass";
+$LOCAL: modal;
+
+$pfe-modal--breakpoint--medium: 640px;
+
+:host {
+	--pfe-modal--MaxWidth: 70vw;
+	--pfe-modal--MinWidth: 50vw;
+	--pfe-modal--MaxWidth--mobile: 94vw;
+	--pfe-modal--MaxHeight: none;
+	--pfe-modal--MinHeight: 40vh;
+	--pfe-modal--Padding--mobile: calc(#{pfe-var(container-padding)} * 1.25) calc(#{pfe-var(container-padding)} * 3) calc(#{pfe-var(container-padding)} * 1.25) calc(#{pfe-var(container-padding)} * 1.25);
+	--pfe-modal--Padding: calc(#{pfe-var(container-padding)} * 2);
+	--pfe-modal__overlay--BackgroundColor: #{pfe-color(overlay)};
+	--pfe-modal--CloseButtonSize: calc(#{pfe-var(ui--element--size)} - 4px);
+	display: block;
+	position: relative;
+}
+
+:host(:not([has_header])) {
+	--pfe-modal--Padding: calc(#{pfe-var(container-padding)} * 2) calc(#{pfe-var(container-padding)} * 4) calc(#{pfe-var(container-padding)} * 2) calc(#{pfe-var(container-padding)} * 2);
+}
+
+.pfe-modal {
+	&__outer {
+		display: flex;
+		position: fixed;
+		height: 100%;
+		width: 100%;
+		top: 0;
+		left: 0;
+		flex-direction: column;
+		align-items: center;
+		justify-content: center;
+		z-index: 2;
+
+		&[hidden] {
+			display: none;
+			z-index: 0;
+		}
+	}
+	&__overlay {
+		position: fixed;
+		height: 100%;
+		width: 100%;
+		top: 0;
+		left: 0;
+		background-color: pfe-local($cssvar: BackgroundColor, $region: overlay);
+		cursor: pointer;
+
+		&[hidden] {
+			display: none;
+		}
+	}
+	&__wrapper {
+		z-index: 2;
+		position: relative;
+		max-width: pfe-local(MaxWidth--mobile);
+		min-width: pfe-local(MinWidth);
+		min-height: pfe-local(MinHeight);
+		margin: 0 auto;
+		box-shadow: pfe-var(box-shadow--lg);
+		background-color: pfe-color(surface--lightest);
+		border-radius: #{pfe-var(ui--border-radius)};
+		overflow-y: auto;
+		overscroll-behavior: contain;
+
+		@media screen and (min-width: $pfe-modal--breakpoint--medium) {
+			max-width: pfe-local(MaxWidth);
+			max-height: pfe-local(MaxHeight);	
+		}
+	}
+	&__container {
```

Comment:
If you add one more div inside the container, then the container could have the border radius, and the inner container could have the scroll bars. This would allow the border radius to apply to the scrollbar side. Right now it has a hard edge on the right side:

![image](https://user-images.githubusercontent.com/456863/62379432-f2439700-b514-11e9-8566-9e0449047ca7.png)


```html
<div class="pfe-modal__container">
   <div class="pfe-modal__container-inner"
   ...
   </div>
</div>
```

```css

--pfe-modal--MaxHeight: 90vh;

.pfe-modal__wrapper {
    z-index: 1;
    min-width: var(--pfe-modal--MinWidth);
    min-height: var(--pfe-modal--MinHeight);
    margin: 0 auto;
    position: absolute;
    top: calc( (100vh - var(--pfe-modal--MaxHeight)) / 2);  // also suggested to set some top/bottom spacers based on the maxHeight var
    bottom: calc( (100vh - var(--pfe-modal--MaxHeight)) / 2); 
    max-width: var(--pfe-modal--MaxWidth);
    max-height: var(--pfe-modal--MaxHeight);
    border-radius: #{pfe-var(ui--border-radius)};
 }

.pfe-modal__container {
    padding: 0;
    max-height: 100%;
    position: relative;
    border: white 1px solid;
    border-radius: #{pfe-var(ui--border-radius)};
    box-sizing: border-box;
    overflow: hidden;
    height: 100vh;
 }

.pfe-modal__container-inner {
    overflow-y: scroll;
    overscroll-behavior: contain;
    position: relative;
    display: block;
    height: calc(100% - var(--pfe-modal--Padding));
    padding: var(--pfe-modal--Padding);
    padding-right: 62px !important;
    margin-bottom: var(--pfe--modal--Padding);
    box-sizing: border-box;
}
```
![image](https://user-images.githubusercontent.com/456863/62379442-fc659580-b514-11e9-81fe-ff020afca910.png)

Reply:
Perfect. Works great now.